api/services/service-router.js

```diff
@@ -9,6 +9,9 @@ module.exports = function(requireSession)
 	            serviceController.savePost(req, res, next);
 	        });
 
+	router.post('/services/delete', requireSession, function(req, res, next) {
```

Comment:
Use the delete method instead of post. And also you can pass the id of the service trough the url. Like this:
```JavaScript
router.delete('/services/:id', requireSession, function(req, res, next) {
	            serviceController.deletePost(req, res, next);
	        });
```
You can access the id using `req.params.id`.